config/initializers/airbrake.rb

```diff
@@ -1,7 +0,0 @@
-#This file is overwritten on deploy
```

Comment:
You might want to check that alphagov-deployment doesn't contain this initialiser to copy like this comment suggests, otherwise deploys will fail (if the file is copied over and `Airbrake` is called).

Reply:
@tijmenb sorry which repo are you referring to with alphagov-deployment ? Is that govuk-app-deployment ? Thanks

Reply:
No worries, I've removed them in https://github.digital.cabinet-office.gov.uk/gds/alphagov-deployment/pull/1347.